docs/spec/v1beta1/imageupdateautomations.md

```diff
@@ -441,6 +464,100 @@ spec:
       branch: auto
 ```
 
+In the following snippet, updates and commits will be made on the `main` branch locally.
+The commits will be then pushed using the `refs/heads/main:refs/heads/auto` refspec:
+
+```yaml
+spec:
+  git:
+    checkout:
+      ref:
+        branch: main
+    push:
+      refspec: refs/heads/main:refs/heads/auto
+```
+
+#### Gerrit
+
+[Gerrit](https://www.gerritcodereview.com/) doesn't behave like a standard Git
+server. Instead of pushing all your changes to a branch in the form of one or
+more commits, you push all your changes as a single commit. This commit must
+have a unique identifier which is different from the commit SHA. Furthermore,
+instead of a Pull Request being opened from one branch to another, you push your
+commit using a refspec: `HEAD:refs/for/main`. Since image-automation-controller
```

Comment:
```suggestion
[Gerrit](https://www.gerritcodereview.com/) operates differently from a
standard Git server. Rather than sending individual commits to a branch,
all changes are bundled into a single commit. This commit requires a distinct
identifier separate from the commit SHA. Additionally, instead of initiating
a Pull Request between branches, the commit is pushed using a refspec:
`HEAD:refs/for/main`. 
```